storops_test/unity/resource/test_filesystem.py

```diff
@@ -245,6 +245,13 @@ def test_modify_success_empty_description(self):
         fs.modify(description='')
         assert_that(fs.description, equal_to(''))
 
+    @patch_rest
+    def test_modify_success_snap_schedule(self):
+        fs = UnityFileSystem(cli=t_rest(), _id='fs_22')
+        snapScheduleParameters = {'snapSchedule': {'id': 'snapSch_1'}}
```

Comment:
snapScheduleParameters need to change to snap_schedule_parameters

Reply:
Done

Reply:
Done